cred-manager/src/sourcecred/service/sourcecred.service.ts

```diff
@@ -62,13 +69,25 @@ export class SourceCredService {
     await this.configureSourcecredGithubPlugin(pluginConfigString);
     await this.startSourceCredCalculation(gitHubToken);
     const credGrainView = await this.loadLocalScInstance();
-    await this.resetSourceCred();
-
     const userCredDtoArray = this.extractUserData(credGrainView);
+
+    const databases = await this.cacheService.getSqlLiteDatabase(
+      this.sourceCredCacheDbPath,
+    );
+
     await this.deleteContribution(teamId);
     const contribution = await this.saveContribution(teamId, credGrainView);
     await this.saveUserScore(contribution.id, userCredDtoArray);
+
+    for (let i = 0; i < databases.length; i++) {
+      const database = databases[i];
+      const credMetrics = await this.loadLocalScMetrics(database);
```

Comment:
We should start breaking down this "calculateCredScores" function, it's getting too big now